### ISSUE TITLE
dostoevsky is not recognized as an internal or external command

### ISSUE THREAD

**Issue by tssvitaly** [bureaucratic-labs/dostoevsky#52]
Hi, perhaps but silly question, but I'm blocked on this step
"First of all, you'll need to download binary model:
$ dostoevsky download fasttext-social-network-model"

after inputting it in cmd I have:
'dostoevsky' is not recognized as an internal or external command,
operable program or batch file.

Thank you for helping

**Comment by polyankaglade** [comment]
same issue, don't know what to do

**Comment by dveselov** [comment]
Hi, are you using Windows?
We didn't support it, try to run `dostoevsky` on Linux or WSL.

**Comment by dveselov** [comment]
Btw, any help with Windows support is appreciated.

**Comment by dveselov** [comment]
Should be fixed in `0.5.0`.
Download models command:
```bash
$ python -m dostoevsky download fasttext-social-network-model
```
